tukey.gemspec

```diff
@@ -21,6 +21,8 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
+  spec.add_dependency 'activesupport'
```

Comment:
Blijft wel een beetje loos voelen om heel activesupport alleen hiervoor aan die Gem toe te voegen. Maar goed, zolang co2m het enige is wat het gebruikt boeit het ook niet veel :p